lib/src/main/java/edu/utexas/tacc/tapis/files/lib/clients/IrodsDataClient.java

```diff
@@ -74,8 +77,20 @@ public IrodsDataClient(@NotNull String oboTenant1, @NotNull String oboUser1, @No
     host = system1.getHost();
     port = system.getPort() == null ? -1 : system.getPort();
     Path tmpPath = Paths.get(rootDir);
+    if(tmpPath.getNameCount() < 1) {
+        String msg = LibUtils.getMsg("FILES_IRODS_ZONE_ERROR", oboTenant, "", oboTenant, oboUser, system.getId(), system.getRootDir());
+        log.error(msg);
+        throw new IOException(msg);
+    }
+    String user = (system.getAuthnCredential() == null) ? null
+            : system.getAuthnCredential().getAccessKey();
+    if(StringUtils.isBlank(user)) {
```

Comment:
This isn't what we discussed here, but I think it's correct (maybe).  We don't want the effective user, we want the irods user from the credentials, right?